src/controller/CHIPDeviceController.h

```diff
@@ -253,6 +253,15 @@ class DLL_EXPORT DeviceController : public Messaging::ExchangeDelegate,
      */
     CHIP_ERROR ServiceEventSignal();
 
+    /**
```

Comment:
This file is not changed but I have worked on csg/test_event_3.1_fabric_id as base so all these changes are showing up in this PR.